modules/compbox/manifests/init.pp

```diff
@@ -473,6 +473,7 @@
             plugin  => nginx,
             domains => [$www_hostname] + $secondary_domains,
             require => Package['nginx', 'certbot'],
+            manage_cron => true,
```

Comment:
We may not be reloading nginx on renewal because of the [manage_install setting](https://forge.puppet.com/modules/puppet/letsencrypt/reference#manage_install) above. We could look at changing that or adding a cron success command.

```suggestion
            manage_cron => true,
            cron_success_command => '/bin/systemctl reload nginx.service',
```

Reply:
My reading is that `manage_install` controls installing of `certbot`, which we do manually.

`certbot` itself should reload `nginx` - without the need for us to do it manually (since it needs to do it anyway to apply the webroot config it needs to do the provision).

I'd really like to try deploying this without this change, confirm the behaviour, and then add the correct fix, rather than a whack-a-mole solution because we're not quite sure how it behaves.

Reply:
That's fair.